### PR TITLE
Fix translation issues with some concatentations.

### PR DESCRIPTION
**Pull request by mrstegeman**
References #2326

### CODE REVIEW

File: static/fluent/en-US/main.ftl [hunk 1]
```diff
@@ -294,8 +294,7 @@ active = Active
 inactive = Inactive
 
 ## Domain Setup
-reclaim-prompt = It looks like you've already registered that subdomain. To reclaim it
-click-here = click here
+tunnel-setup-reclaim-domain = It looks like you've already registered that subdomain. To reclaim it <a data-l10n-name="tunnel-setup-reclaim-domain-click-here">click here</a>.
```
Comment (flodolo) [inline static/fluent/en-US/main.ftl]: Apostrophes ;-)

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: That fix is in a separate PR.

Comment (flodolo) [reply static/fluent/en-US/main.ftl]: But this a string that it's not available in that PR yet, isn't it?

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: No, the strings here are not new, just concatenation properly.

Comment (flodolo) [reply static/fluent/en-US/main.ftl]: Right, but merging the PR with the apostrophes fix won't have effect on this one. Rebasing will remove the two existing strings, you'll still need to manually update this string with the correct apostrophe.

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: Updated.

**Pull request merged**